<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveAsUI</class>
 <widget class="QWidget" name="SaveAsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>55</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="saveASLabel">
       <property name="text">
        <string>Save As: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="saveASEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="saveASButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
